google/validation.go

```diff
@@ -0,0 +1,24 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"regexp"
+)
+
+func validateGCPName(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Or, better suggestion - make the other function return (ws []string, errors []error) (not a func)